package/touchui/touchui.mk

```diff
@@ -0,0 +1,43 @@
+################################################################################
+#
+# touchui
+#
+################################################################################
+
+TOUCHUI_VERSION = 190827e49b6ae5279b5e9b0bcf9cb5b772278e06
+TOUCHUI_SITE = $(call github,ski7777,TouchUI,$(TOUCHUI_VERSION))
```

Comment:
Pulling TouchUI out of ftcommunity-TXT just to put it into another repo without any connection to upstream does not make sense. 

Please use the original TouchUI repo from @harbaum here.